src/main/java/eu/deltasource/internship/hotelmanagementsystem/hotel/service/domain/commodities/Toilet.java

```diff
@@ -1,20 +1,25 @@
-package commodities;
+package eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities;
 
+/**
+ * Toilet is part of the set of commodities
+ */
 public class Toilet extends AbstractCommodity {
 
     private boolean hasWaterSavingTechnology;
     private String color;
 
-    public Toilet(int inventoryNumber, boolean hasWaterSavingTechnology, String color) {
-        super(inventoryNumber);
+    /**
+     * Parametrized constructor
```

Comment:
The "Parametrized constructor" is not very helpful, describing the parameter's in this case is good enough